earthly-next

```diff
@@ -0,0 +1 @@
+d4e630c482bde6e4f6da3d2257ee6e8d9955f180
```

Comment:
note that we'll have to start updating this file when we do releases for the `-ticktock` version.